tests/test_api_lib_utils.py

```diff
@@ -75,8 +76,12 @@ def test_03_verify_auth_token(self):
                                          "resolver": "resolverX"},
                                 key=key,
                                 algorithm="RS256")
-        self.assertRaisesRegexp(AuthError, "The username hanswurst is not allowed to impersonate via JWT.",
-                                verify_auth_token, auth_token=auth_token, required_role="user")
+        with warnings.catch_warnings():
```

Comment:
Tell me more...

Reply:
Well, it catches warnings :-) ... and ignores them, in this case a `DeprecationWarning` emitted from `pytest`. In the end i did not wrap all emitters of deprecation warnings.